lib/lotus/model/adapters/sql/query.rb

```diff
@@ -663,6 +660,11 @@ def apply(query)
             end
           end
 
+          def _push_to_conditions(condition_type, condition)
+            raise ArgumentError, 'You need to specify a condition.' unless condition
```

Comment:
If we remove this raise, then `conditions.push` will add a condition_type with a nil condition, this maybe can be weird and make bugs in the code, because condition can be a condition or a block